README.md

```diff
@@ -1,5 +1,5 @@
 # Developer Portal
-[![CI](https://github.com/pagopa/developer-portal/actions/workflows/ci.yaml/badge.svg)](https://github.com/pagopa/developer-portal/actions/workflows/ci.yaml)
+[![Code Review](https://github.com/pagopa/developer-portal/actions/workflows/code_review.yaml/badge.svg)](https://github.com/pagopa/developer-portal/actions/workflows/code_review.yaml)
```

Comment:
What about removing the badge? I don't think this badge is useful

Reply:
Yeah, it makes sense actually! I'm going to remove it.